test/test.h

```diff
@@ -96,7 +96,7 @@ void assert_cbor2json(char const *name, char const *json_value, uint16_t id, cha
 #endif
 
 #define ASSERT_MSG(assert_func) \
-        _ASSERT_MSG("Assertion triggered by " assert_func " at #" STRINGIFY(__LINE__) " " __FILE__)
+        _ASSERT_MSG("Assertion triggered by " assert_func " at " __FILE__ ":" STRINGIFY(__LINE__))
```

Comment:
File path names can become very long. It is hard to spot the line number after such a long file path.

Reply:
I changed this because VS code allows to jump to the line of code if it's in the form of path-to-file.c:123 with Ctrl + click. But it was hidden in an unrelated commit anyway, which is not good practice. Reverted it to previous version for now.